### ISSUE TITLE
Reflections.collect() logs an ERROR, which isn't one

### ISSUE THREAD

**Issue by GoogleCodeExporter** [NickstaDB/reflections#88]
```
What steps will reproduce the problem?
1.using: Reflections reflections = isProduction() ? Reflections.collect() : new 
Reflections("your.package.here");
and the maven plugin
2. in development and in production it works! But in Production the 
Reflections.collect() logs an ERROR: "given scan urls are empty. set urls in 
the configuration"
3.

What is the expected output? What do you see instead?
Expected: No error log if it works 
Instead: ERROR: "given scan urls are empty. set urls in the configuration"

What version of the product are you using? On what operating system?
0.9.5 / windows and linux

Please provide any additional information below.


```

Original issue reported on code.google.com by `mario.p...@gmail.com` on 26 Sep 2011 at 8:50

**Comment by GoogleCodeExporter** [comment]
```
Hit the same problem.
This happens when no URLs are given because data is being loaded from a 
Serializer.
Around -RC2 there was a constructor for this use case, which initialized the 
Store but didn't perform the scan().
```

Original comment by `kronos...@gmail.com` on 2 Nov 2011 at 4:44

**Comment by GoogleCodeExporter** [comment]
```
fixed on 0.9.7-RC1 and trunk. comments?
```

Original comment by `ronm...@gmail.com` on 2 May 2012 at 3:55
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
fixed 0.9.7
```

Original comment by `ronm...@gmail.com` on 22 May 2012 at 7:47
- Changed state: **Fixed**
